lib/pure/osproc.nim

```diff
@@ -763,6 +763,10 @@ elif not defined(useNimRtl):
     var sysArgsRaw: seq[string]
     if poEvalCommand in options:
       sysCommand = "/bin/sh"
+      when defined(useShPath):
+        var shPath {.strdefine.}: string = nil
```

Comment:
@Araq, thanks for the suggestion did exactly that :) 3141048b53f63e7789c829a56bfee3113e46f60e